src/commonMain/kotlin/com/charleskorn/kaml/YamlConfiguration.kt

```diff
@@ -67,3 +69,20 @@ public enum class SequenceStyle {
      */
     Flow
 }
+
+public enum class MultiLineStringStyle {
+    Literal,
+    DoubleQuoted,
+    SingleQuoted,
+}
+
+public enum class QuotedScalarStyle {
+    DoubleQuoted,
+    SingleQuoted;
+
+    public val multiLineStringStyle: MultiLineStringStyle
+        get() = when(this) {
+            DoubleQuoted -> MultiLineStringStyle.DoubleQuoted
+            SingleQuoted -> MultiLineStringStyle.SingleQuoted
+        }
+}
```

Comment:
Should we just have one type to cover both of these? There's no harm in allowing someone to emit single line strings with the literal style.

Reply:
I would REALLY prefer this not to be the case, it really reduces the readability of the produced YAML and it almost doubles in length of the document when every single strings is in block literal format